src/components/Footer.js

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+import IconLink from "@components/typography/IconLink";
+import { ThemeContext } from "@contexts/theme/ThemeContext";
+import { useTheme } from "@hooks/useTheme";
+import facebookIcon from "@images/icons/facebook.svg";
+import githubIcon from "@images/icons/github.svg";
+import linkedInIcon from "@images/icons/linkedin.svg";
+import { footerStyles } from "@styles/FooterC&S";
+
+const Footer = () => {
+  const { themeState } = useTheme(ThemeContext)
+
+  return (
+    <footer className={footerStyles[`${themeState.theme}`].footer}>
+      <h4 className={footerStyles[`${themeState.theme}`].h4}>@2022 Wiktor Rudzki</h4>
+      <div className={footerStyles[`${themeState.theme}`].paddingMarginZero}>LOGO</div>
+      <ul className={footerStyles[`${themeState.theme}`].icons}>
+        <IconLink
+          url="https://www.facebook.com/wiktor.rudzki.73"
+          alt="Find me on Facebook!"
+          icon={facebookIcon}
+          style={`${footerStyles[`${themeState.theme}`].icon} ${footerStyles[`${themeState.theme}`].paddingMarginZero}`}
```

Comment:
Tutaj na pewnoe będziemy coś jeszcze zmieniać, ale muszę się zastanowić, jakby to lepiej zrobić